resources/sass/default.style.scss

```diff
@@ -7,6 +7,7 @@
 // Component
 // -----------------------------------------------------------------------------
 @import "components/my.component";
```

Comment:
I could do some cleaning up of files we dont use, what do you think ?

Reply:
We should do it in a separate issue, weird to see here "my.component".